src/classes/issues-processor.ts

```diff
@@ -121,6 +124,13 @@ export class IssuesProcessor {
     }
 
     for (const issue of issues.values()) {
+      const startMessage = LoggerService.white(
```

Comment:
I wonder if the start/end group should live in the `IssueLogger`? Then the start group could be part of the initial creation of the logger. 

For the `endGroup` maybe the issueLogger could wrap the `core.endGroup`. 

Also I believe there is a mechanism in the logger for using a `$$type` which will print out either issue or pull request.

Reply:
However, if I have the `startGroup` in the constructor, then many more groups are created than we actually need, because the `IssueLogger` is always recreated. So I would only create a method `startGroup` and `endGroup` in the logger.

Reply:
Isn't an `IssueLogger` created for each issue?